lists/lt/lt-vmi.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": {
+    "en": "Lithuanian State Tax Inspectorate",
+    "local": "Lietuvos Valstybinė Mokesčių Inspekcija"
+  },
+  "url": "https://www.vmi.lt/",
+  "description": {
+    "en": "The State Tax Inspectorate is Lithuania's institution responsible for tax administration. The tasks of the State Tax Inspectorate include: helping taxpayers and withholding agents to exercise their rights and obligations; implement tax laws; ensuring the payment of taxes to the budget; ensuring the efficient work of the Bank of Lithuania by coordinating, controlling and methodically managing it. "
```

Comment:
I think we need to detail the actual identifier here that this entry relates to. Esp since 'Identification number', 'VAT code' and 'Individual activity certificate number' are all mentioned on the search page.

The title of this entry might also need to be updated to reflect this?

Reply:
I've updated the text to include your suggestion - does this seem ok to you? Thanks for catching the grammar error